### ISSUE TITLE
xrdcp: confused error message after (local, destination) ENOSPC

### ISSUE THREAD

**Issue by jmuf** [xrootd/xrootd#1776]
The following error message when running out of local disk space could perhaps be improved (from a UI perspective):
```
~$ xrdcp root://...tar /tmp/
[4.086GB/74.29GB][  5%][==>                                               ][190.2MB/s]  
Run: [ERROR] Server responded with an error: [3005] no error (destination)
```
* "Server responded": this error was from the client, not the server
    * "no error" string ? perhaps an artifact of the above..
    * the non-translated '3005' error code (`kXR_FSError`) adds further confusion
*  the actual local error message ENOSPC/"No space left on device" is missing

```
~$ df -h /tmp
Filesystem      Size  Used Avail Use% Mounted on
/dev/vda2       4.6G  4.3G     0 100% /tmp
~$ rpm -qf `which xrdcp`
1:xrootd-client-5.4.3-1.el8.x86_64
```

**Comment by simonmichal** [comment]
@jmuf : thanks for reporting this problem, it shall be fixed!

**Comment by simonmichal** [comment]
It should be fixed in 37bfb705b2cde9c707251d7793cb303dabb60878, do you want to give it a try before I close this issue?

**Comment by simonmichal** [comment]
I think we can close this one.
